.gitignore

```diff
@@ -3,3 +3,4 @@ yarn-error.log
 
 # macOS
 .DS_Store
+.yarnrc
```

Comment:
You can also check-in this file. Then the registry setting is available for everyone, no matter what the developer's user config is